go/host/host.go

```diff
@@ -716,6 +699,47 @@ func (a *Node) monitorBlocks() {
 	subs.Unsubscribe()
 }
 
+func (a *Node) catchupMissedBlocks(lastKnownBlkHash gethcommon.Hash) {
+	var lastBlkNumber *big.Int
+	var reingestBlocks []*types.Block
+
+	// get the tip block
+	lastBlk, err := a.ethClient.BlockByNumber(lastBlkNumber)
+	if err != nil {
+		log.Panic("catching up on missed blocks, unable to fetch tip block - reason: %s", err)
+	}
+
+	if lastBlk.Hash().Hex() == lastKnownBlkHash.Hex() {
+		// no new blocks have been issued
+		return
+	}
+	reingestBlocks = append(reingestBlocks, lastBlk)
+
+	// iterate from the tip (last block produced) to the last block ingested by the node
+	for lastBlk.Hash().Hex() != lastKnownBlkHash.Hex() {
+		blockParent, err := a.ethClient.BlockByHash(lastBlk.ParentHash())
+		if err != nil {
+			log.Panic("catching up on missed blocks, could not fetch block's parent with hash %s. Cause: %s", lastBlk.ParentHash(), err)
+		}
+
+		reingestBlocks = append(reingestBlocks, blockParent)
+		lastBlk = blockParent
+	}
+
+	// make to have the last ingested block available
```

Comment:
this sentence doesn't quite make sense, think maybe words have gone missing after 'make'?

Reply:
```suggestion
	// make sure to have the last ingested block available for ingestion (because we always ingest ( blk, blk_parent)
```